File: src/test/java/ru/nucodelabs/ShiraPicket.java
package ru.nucodelabs;

import ru.nucodelabs.data.ves.Picket;
import ru.nucodelabs.gem.app.io.SonetImportManager;

import java.io.File;

public class ShiraPicket {
    private ShiraPicket() {

    }

    /**
     * Делает готовый Shira пикет, с готовой моделью
     *
     * @return Пикет Shira
     */
    public static Picket getPicket() throws Exception {
        File file_stt = new File("data/SHIRA.STT");

        File file_exp = new File("data/SHIRA.EXP");

        File file_mod = new File("data/SHIRA_M2.mod");

        var picket = Picket.defaultValue();
        SonetImportManager sonetImportManager = SonetImportManager.create();
        picket = sonetImportManager.loadNameAndExperimentalDataFromEXPFile(file_exp, picket);
        picket = sonetImportManager.loadModelDataFromMODFile(file_mod, picket);
        return picket;
    }
}

File: src/test/java/ru/nucodelabs/ShiraPicket.kt
package ru.nucodelabs

import ru.nucodelabs.gem.app.io.SonetImportManager.Factory.create
import ru.nucodelabs.geo.ves.Picket
import java.io.File

object ShiraPicket {
    /**
     * Делает готовый Shira пикет, с готовой моделью
     *
     * @return Пикет Shira
     */
    @JvmStatic
    @get:Throws(Exception::class)
    val picket: Picket
        get() {
            val expFile = File("data/sonet_examples/SHIRA.EXP")
            val modFile = File("data/sonet_examples/SHIRA_M2.mod")
            val sonetImportManager = create()
            val picket = sonetImportManager.fromEXPFile(expFile)
            return picket.copy(modelData = sonetImportManager.fromMODFile(modFile))
        }
}